infra/modules/azure_monorepo_single_env_starter_pack/github_repository.tf

```diff
@@ -0,0 +1,41 @@
+resource "github_repository" "this" {
+  name        = var.repository.name
+  description = var.repository.description
+  topics      = var.repository.topics
+
+  visibility = "public"
+
+  allow_auto_merge            = true
+  allow_rebase_merge          = false
+  allow_merge_commit          = false
+  allow_squash_merge          = true
+  squash_merge_commit_title   = "PR_TITLE"
```

Comment:
```suggestion
  squash_merge_commit_title   = "PR_TITLE"
  squash_merge_commit_message = "COMMIT_MESSAGES"
```

Reply:
Any particular reason? I don't like it because it's completely useless